backend/uclapi/oauth/views.py

```diff
@@ -456,17 +459,32 @@ def token(request):
 @uclapi_protected_endpoint(personal_data=True)
 def userdata(request, *args, **kwargs):
     token = kwargs['token']
+    print("Checking student status")
+    try:
+        get_student_by_upi(
+            token.user.employee_id
+        )
+        is_student = True
```

Comment:
Not a big fan of this `try: flag = True except: flag = False` pattern but for now it suffices